src/inc/internal/BundleWriterHelper.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+
+namespace MSIX {
+
+    struct PackageInfo
+        {
+            APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE type;
+            UINT64 version;

```

Comment:
>UINT64 [](start = 12, length = 6)

try using the std types

Reply:
Updated to use std::string and std::uint64_t.